funcx_endpoint/funcx_endpoint/executors/process_pool/process_pool.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import logging
+import multiprocessing
+import uuid
+from concurrent.futures import Future
+from concurrent.futures import ProcessPoolExecutor as NativeProcessPoolExecutor
+
+from funcx_common import messagepack
+from funcx_common.messagepack.message_types import Task
+from parsl.executors.status_handling import NoStatusHandlingExecutor
+
+from funcx_endpoint.executors.execution_helper.helper import execute_task
+
+logger = logging.getLogger(__name__)
+
+
+class ProcessPoolExecutor(NoStatusHandlingExecutor):
+    def __init__(self, *args, **kwargs):
+        self.executor = NativeProcessPoolExecutor(*args, **kwargs)
+        self.results_passthrough: multiprocessing.Queue | None = None
+
+    def start(self, results_passthrough: multiprocessing.Queue, run_dir=None) -> None:
+        """
+
+        Parameters
+        ----------
+        results_passthrough: Queue to which packed results will be posted
+        run_dir Not used
+
+        Returns
+        -------
+
+        """
+        self.results_passthrough = results_passthrough
+        return
+
+    def _future_done_callback(self, future: Future):
+        """Callback to post result to the passthrough queue
+
+        Parameters
+        ----------
+        future: Future for which the callback is triggerd
+
+        Returns
+        -------
+        None
+
+        """
+        logger.warning(f"[YADU] task:{future.task_id} completed")
+        self.results_passthrough.put(future.result())
```

Comment:
Wooo!

Reply:
I do try to put a personal touch on the warnings :)